### DISCUSSION TITLE
全局管理角色 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1007] [Announcements]
# 全局管理角色 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/ghippo/user-guide/access-control/global/

**Comment by liyuerich** [comment]
系统还有这个角色：Workspace and Folder Owner

**Reply by yitingdc** [reply]
已经创建 https://gitlab.daocloud.cn/ndx/ghippo/-/issues/632 去跟踪

**Reply by WANG0608GitHub** [reply]
> 系统还有这个角色：Workspace and Folder Owner

Fixed in PR #1612
